src/js/views/maps/ExpansionPanelView.js

```diff
@@ -134,7 +130,7 @@ define(
           this.getContent().append(this.contentViewInstance.el);
 
           if (this.templateVars.isSvgIcon) {
-            this.insertIcon(this.icon);
+            this.insertIcon(this.templateVars.icon);
           }
```

Comment:
this is covered in the template. no need to insert in js anymore

Reply:
Done